rpm/SPECS/iotajson.spec

```diff
@@ -172,7 +172,7 @@ fi
 * Tue Aug 13 2019 Fermin Galan <fermin.galanmarquez@telefonica.com> 1.11.0
 - Set Nodejs 8 as minimum version in packages.json (effectively removing Nodev6 from supported versions)
 - Add: Reconnect when MQTT closes connection (including mqtt retries and keepalive conf options)
-- Upgrade iotagent-node-lib dependency from 2.8.1 to 2.9.0
+- Upgrade iotagent-node-lib dependency from 2.9.0 to 2.10.0
```

Comment:
This seems to be already in master (see current https://github.com/telefonicaid/iotagent-json/blob/master/rpm/SPECS/iotajson.spec). Maybe the branch is out of sync and once gets updated with mater this diff will disappear.

Reply:
I've rebased the change to clean up the merge conflict. Fixed 998c6635